Formula/materialized.rb

```diff
@@ -6,8 +6,8 @@ class Materialized < Formula
   head "https://github.com/MaterializeInc/materialize.git"
 
   bottle do
-    root_url "https://downloads.mtrlz.dev"
-    sha256 "a14fb6a49a4f5e52e08ea37e02f0abf7f35889f92ba33017e428fbeac41fca20" => :high_sierra
+    root_url "https://packages.materialize.io/homebrew"
+    sha256 "84f9b4c05680e3e84f0f1f0af6a0dafc293624f3cccee8faabb7e95096dfa8a9" => :high_sierra
```

Comment:
Hmm, why did the SHA change? Bottle building should be entirely deterministic given an input binary tarball.

Reply:
I don't know, but I suspect it is related to the fact that CI is failing :) investigating.

Reply:
We'll have to solve this to keep CI happy. CI and I agree that the tarball should have the same SHA as before, 84f9b4c05680e3e84f0f1f0af6a0dafc293624f3cccee8faabb7e95096dfa8a9, even with your changes to the script.

Are you using a different version of gnu tar? CI and I have 1.32.